### PR TITLE
Replace json slurper with readJSON

### PR DESCRIPTION
**Pull request by johnbieren**
https://github.com/CentOS-PaaS-SIG/ci-pipeline/pull/613

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

**Comment by 14rcole** [comment]
LGTM

File: src/org/centos/pipeline/PipelineUtils.groovy [hunk 1]
```diff
@@ -1371,7 +1370,8 @@ def injectCIMessage(String prefix, def ciMessage) {
         def new_key = key.replaceAll('-', '_')
         if (value instanceof groovy.json.internal.LazyMap) {
             injectCIMessage("${prefix}_${new_key}", value)
-        } else if (value instanceof  java.util.ArrayList) {
+        } else if (value instanceof java.util.ArrayList || value instanceof net.sf.json.JSONArray) {
```
[the remark this answers is not in the document]
Comment (johnbieren) [reply src/org/centos/pipeline/PipelineUtils.groovy]: Yes, but https://github.com/CentOS-PaaS-SIG/ci-pipeline/pull/613 broke the pipeline. I had to revert it. See https://github.com/CentOS-PaaS-SIG/ci-pipeline/pull/638

I needed the upstream pipeline to work while I tried to figure it out

**Comment by joejstuart** [comment]
lgtm

**Comment by johnbieren** [comment]
[merge]

**Pull request merged**